src/Controller/MapController.php

```diff
@@ -5,16 +5,44 @@
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+use App\Repository\FablabRepository;
+use App\Entity\Fablab;
+use maxh\Nominatim\Nominatim;
 
+/**
+ * @Route("/map")
+*/
 class MapController extends AbstractController
 {
     /**
-     * @Route("/map", name="map")
+     * @Route("/", name="map")
      */
     public function index(): Response
     {
+        $fablabRepository = $this->getDoctrine()
```

Comment:
your variable doesn't contain a repository, but a collection of fablab: something like `$fablabs` would better suit

Reply:
done.